rainbow-cake-koin/src/main/java/co/zsmb/rainbowcake/koin/RainbowCakeFragment.kt

```diff
@@ -18,13 +21,41 @@ import org.koin.core.qualifier.named
 public inline fun <F : RainbowCakeFragment<VS, VM>, VS, reified VM : RainbowCakeViewModel<VS>> F.getViewModelFromFactory(
         scope: ViewModelScope = Default
 ): VM {
+    return getFragmentViewModel(scope)
+}
+
+/**
+ * Uses the global Koin instance to fetch the appropriate ViewModel instance for the Fragment.
+ *
+ * @param scope The scope that the ViewModel should be fetched from and exist in.
+ *              See [ViewModelScope] for details.
+ */
+public inline fun <F : RainbowCakeDialogFragment<VS, VM>, VS, reified VM : RainbowCakeViewModel<VS>> F.getViewModelFromFactory(
+        scope: ViewModelScope = Default
+): VM {
+    return getFragmentViewModel(scope)
+}
+
+/**
+ * Uses the global Koin instance to fetch the appropriate ViewModel instance for the Fragment.
+ *
+ * @param scope The scope that the ViewModel should be fetched from and exist in.
+ *              See [ViewModelScope] for details.
```

Comment:
Is this indentation here for any particular reason?

Reply:
It makes this line continue the description for the `scope` param.

![image](https://user-images.githubusercontent.com/12054216/100762778-acdaf680-33f4-11eb-9191-0074883218c8.png)